api/config/v1alpha1/shared_types.go

```diff
@@ -83,6 +83,12 @@ type KubernetesPodSpec struct {
 	// If specified, the pod's tolerations.
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+
+	// Volumes that can be mounted by containers belonging to the pod.
+	// More info: https://kubernetes.io/docs/concepts/storage/volumes
+	//
+	// +optional
+	Volumes []corev1.Volume `json:"volumes,omitempty"`
```

Comment:
should this be at the top level instead of under `pod` ?

Reply:
I think should keep same level like `Affinity` at `Deployment` yaml